src/web/controllers/admin/index_admin_controller.py

```diff
@@ -240,6 +242,9 @@ def _admin_render(
             console_level_infos[value]['name'] = name
 
         context = web_context.template_context | {
+            'application_federation': sharly_chess_config.federation
+            if sharly_chess_config.stored_config.federation
+            else None,
```

Comment:
No need to pass an additional variable.
```suggestion
            'application_federation': sharly_chess_config.federation
            if sharly_chess_config.stored_config.federation
            else None,
```